Cargo.toml

```
@@ -21,26 +21,24 @@ maintenance = { status = "actively-developed" }
 all-features = true
 targets = ["x86_64-unknown-linux-gnu", "i686-pc-windows-msvc", "x86_64-apple-darwin"]
 
-[workspace]
-members = [
-    "ffi",
-]
-
 [features]
 default = ["passthru", "socketcan"]
 socketcan = ["dep:socketcan-isotp", "dep:socketcan"]
 passthru = ["dep:libloading", "dep:shellexpand", "dep:winreg", "dep:serde_json", "dep:j2534_rust"]
 
-
 [dependencies]
-auto_uds = "0.4"
+#automotive_diag = { version = "0.1", path = "../automotive_diag" }
+automotive_diag = { version = "0.1", git = "https://github.com/nyurik/automotive_diag.git", branch = "main" }
```

Not a biggie, but keeps things cleaner
```suggestion
automotive_diag = "0.1"
```